tests/data_context/cloud_data_context/test_checkpoint_crud.py

```diff
@@ -118,53 +117,53 @@ def _mocked_get_response(*args, **kwargs):
                         "organization_id": f"{organization_id}",
                         "updated_at": "2022-08-02T17:55:45.107550",
                     },
-                    "id": contract_id,
+                    "id": checkpoint_id,
                     "links": {
-                        "self": f"/organizations/{organization_id}/contracts/{contract_id}"
+                        "self": f"/organizations/{organization_id}/checkpoints/{checkpoint_id}"
                     },
                     "relationships": {
```

Comment:
note: I don't think you're using these, but we're going to drop `relationships` in all our future APIs